### PR TITLE
use js-server-sdk instead of openapi

### PR DESCRIPTION
**Pull request by karkakol**
Use js-server-sdk instead of openapi generation

### CODE REVIEW

File: example/webdriverio-test/package.json [hunk 1]
```diff
@@ -18,7 +18,8 @@
     "appium-xcuitest-driver": "^5.11.0",
     "ts-node": "^10.9.1",
     "typescript": "^5.3.2",
-    "webdriverio": "^8.24.2"
+    "webdriverio": "^8.24.2",
+    "@fishjam-dev/js-server-sdk": "^0.1.0"
```
Comment (mironiasty) [inline example/webdriverio-test/package.json:22]: When you modify this file manually, please make sure you are keeping alphabetical order (as `yarn` does).

File: example/webdriverio-test/test/specs/test.e2e.ts [hunk 2]
```diff
@@ -8,14 +8,16 @@ import {
...
-  Configuration,
   ConfigurationParameters,
+  Configuration,
+  RoomApiFp,
+  AddPeerRequest,
   PeerDetailsResponseData,
   Room,
-  RoomApiFp,
-} from '../../server-api';
+} from '@fishjam-dev/js-server-sdk';
```
Comment (mironiasty) [inline example/webdriverio-test/test/specs/test.e2e.ts:19]: We should verify why our CI checks allowed this incorrect import order. I think it is defined as one of eslint/prettier rules 🤔

**Review by mironiasty — APPROVED** [review]
Two small nits about ordering.

**Comment by graszka22** [comment]
LGTM

File: example/webdriverio-test/test/specs/test.e2e.ts (hunk 2 above)
Comment (karkakol) [reply example/webdriverio-test/test/specs/test.e2e.ts:19]: I do not think it was defined anywhere.

**Pull request merged**